src/test/data/JsonAddressBookStorageTest/invalidAndValidPersonAddressBook.json

```diff
@@ -1,13 +1,20 @@
 {
-  "persons": [ {
-    "name": "Valid Person",
-    "phone": "9482424",
-    "email": "hans@example.com",
-    "address": "4th street"
-  }, {
-    "name": "Person With Invalid Phone Field",
-    "phone": "948asdf2424",
-    "email": "hans@example.com",
-    "address": "4th street"
-  } ]
+  "persons": [
+    {
+      "name": "Valid Person",
+      "employeeId": "1",
+      "phone": "9482424",
+      "email": "hans@example.com",
+      "position": "Worker",
+      "address": "4th street"
+    },
+    {
+      "name": "Person With Invalid Phone Field",
+      "employeeId": "aaa",
```

Comment:
Change to a valid employee ID. Maybe add more cases also.

Reply:
Okay, updated in commit `7627bbe`.